desk/app/reel.hoon

```diff
@@ -102,6 +108,13 @@
     ``reel-metadata+!>(metadata)
   ==
 ::
-++  on-arvo   on-arvo:def
+++  on-arvo
+  |=  [=wire sign=sign-arvo]
+  ^-  (quip card:agent:gall _this)
+  ?>  ?=([%set-ship ~] wire)
+  ?>  ?=([%khan %arow *] sign)
+  ?:  ?=(%.n -.p.sign)
+    ((slog leaf+<p.p.sign> ~) `this)
```

Comment:
This will print the error very poorly, because it'll print the `tang` structure instead of printing just the tang.

`p.p.sign` is a `[term tang]`, and since `term` is a type of `tank`, it's just a `tang`.  So it could be `(slog p.p.sign)`, except that we should always always always include a tag on these printfs (I've spent way too many hours searching for unlabeled printfs).  Eg: `(slog 'reel: fetch bait ship failed' p.p.sign)`.